### PR TITLE
Stellar client sell orders should not require stellar_sdk arguments

### PR DESCRIPTION
**Pull request by robvanmieghem**
Solves #258

### CODE REVIEW

File: jumpscale/clients/stellar/stellar.py [hunk 1]
```diff
@@ -800,7 +802,7 @@ def check_is_payment_transaction(self, transaction_hash):
         results = response["_embedded"]["records"][0]
         return results["type"] == "payment"
 
-    def get_asset(self, code="TFT", issuer=None) -> stellar_sdk.Asset:
+    def _get_asset(self, code="TFT", issuer=None) -> stellar_sdk.Asset:
```
Comment (xmonader) [inline jumpscale/clients/stellar/stellar.py:805]: Hiding this function breaks stellar and automation tests https://github.com/threefoldtech/js-sdk/search?q=get_asset

Comment (robvanmieghem) [reply jumpscale/clients/stellar/stellar.py:805]: Good catch, adapted both now. When having a wallet with sdk abstraction, there should be no need to use the sdk types.

**Review by waleedhammam — APPROVED** [review]

File: jumpscale/clients/stellar/stellar.py (hunk 1 above)
Comment (waleedhammam) [reply jumpscale/clients/stellar/stellar.py:805]: If you could run black on the changed files too would be great, to make the ci pass

Comment (robvanmieghem) [reply jumpscale/clients/stellar/stellar.py:805]: Thank you @waleedhammam  and sorry I did not do that before, automated black formatting in my editor was broken.

**Review by xmonader — APPROVED** [review]

**Pull request merged**